src/typings/Types.ts

```diff
@@ -113,6 +113,8 @@ export interface SubTokenModifier {
   pieceType: SubTokenType;
   anchor: number;
   active: number;
+  includeAnchor?: boolean;
+  includeActive?: boolean;
```

Comment:
making these optional for now for backwards compatibility.  Also, I went with "include" instead of "exclude" because I prefer positive booleans, otherwise it's a double negative when they're false which is a bit more cognitive load

my one hesitation with using "include" instead of "exclude" is that while these are optional, we'll be defaulting to `true`, which is always a bit jarring cc/ @AndreasArvidsson

Reply:
I will always prefer that optional booleans default to false. That why they were defined as excluding.

Reply:
Yes but I would prefer to make these not optional. The reason they're optional is for backwards compatibility. Hmm. I'm not seeing a way to make them backwards compatible while making them positive. Any ideas?

Reply:
In Infer full targets just do.
Includeachor = target.includeanchor ?? true

Reply:
Wait isn't that defaulting to `true`?

Reply:
Yes and that is how it is today